app/models/miq_worker/runner.rb

```diff
@@ -374,6 +375,32 @@ def heartbeat_to_drb
   def heartbeat_to_file(timeout = nil)
     timeout ||= worker_settings[:heartbeat_timeout] || Workers::MiqDefaults.heartbeat_timeout
     File.write(@worker.heartbeat_file, (Time.now.utc + timeout).to_s)
+
+    get_messages.each { |msg, *args| process_message(msg, *args) }
```

Comment:
Do we see a need going through all of the extra work of looping through an array of messages if there will only ever be one type we will be handling via memcache?  We could simplify `get_messages` to something like this:

```ruby
def sync_config_if_needed
  @my_last_config_change ||= Time.now.utc
  last_config_change = server_last_change(:last_config_change)
  
  if last_config_change && last_config_change > @my_last_config_change
    _log.info("#{log_prefix} Configuration has changed, New TS: #{last_config_change}, Old TS: #{@my_last_config_change}")
    sync_config
    @my_last_config_change = Time.now.utc
  end
end
```

And just change this line to `sync_config_if_needed`?  No new array needed, and we will just exit early if the `if` in the above isn't satisfied.